src/containers/PDisk/PDisk.tsx

```diff
@@ -46,20 +50,37 @@ export function PDisk() {
     }, [dispatch]);
 
     const fetchData = useCallback(
-        (isBackground: boolean) => {
+        (isBackground?: boolean) => {
```

Comment:
```suggestion
        async (isBackground?: boolean) => {
```

or return `Promise.resolve(undefined);`